packages/terra-infinite-list/src/InfiniteList.jsx

```diff
@@ -499,6 +499,7 @@ class InfiniteList extends React.Component {
     if (role && role.length > 0 && role !== 'none') {
       attrSpread.role = role;
     }
+    attrSpread.tabIndex = '0';
```

Comment:
Is there a test to validate this change?

Reply:
Added jest in 4bc7213

Reply:
Added jest in 4bc7213